base/Makefile

```diff
@@ -110,5 +81,3 @@ clean:
 	rm -f $(BUILDDIR)/fenv_constants.jl
 	rm -f $(BUILDDIR)/uv_constants.jl
 	rm -f $(BUILDDIR)/file_constants.jl
-	rm -f $(BUILDDIR)/version_git.jl
-	rm -f $(BUILDDIR)/version_git.jl.phony
```

Comment:
Could we keep theese rules, an rather add a comment that they are obsolete (so that we might find and delete them in a future cleanup). Many users following Julia will get confused when new files pops up in `git status` now that they are not ignored in `.gitiginore`, and it would be nice if `make clean` fixed it.